### PR TITLE
Add german readme

### PR DESCRIPTION
**Pull request by X00LA**
## Checklist:

- [X] I have read the [Contributing guidelines](https://github.com/SillyTavern/SillyTavern/blob/release/CONTRIBUTING.md).

### CODE REVIEW

File: .github/readme-de_de.md [hunk 1]
```diff
@@ -0,0 +1,383 @@
...
+[![GitHub Stars](https://img.shields.io/github/stars/SillyTavern/SillyTavern.svg)](https://github.com/SillyTavern/SillyTavern/stargazers)
+[![GitHub Forks](https://img.shields.io/github/forks/SillyTavern/SillyTavern.svg)](https://github.com/SillyTavern/SillyTavern/network)
+[![GitHub Issues](https://img.shields.io/github/issues/SillyTavern/SillyTavern.svg)](https://github.com/SillyTavern/SillyTavern/issues)
+[![GitHub Pull Requests](https://img.shields.io/github/issues-pr/SillyTavern/SillyTavern.svg)](https://github.com/SillyTavern/SillyTavern/pulls)
+
+</div>
+
+---
+
+SillyTavern bietet eine einheitliche Benutzeroberfläche für viele LLM-APIs (KoboldAI/CPP, Horde, NovelAI, Ooba, Tabby, OpenAI, OpenRouter, Claude, Mistral und mehr), ein mobilfreundliches Layout, einen Visual-Novel-Modus, die Integration von Automatic1111 & ComfyUI API zur Bilderzeugung, TTS, WorldInfo (Lorebooks), anpassbare UI, automatische Übersetzung, mehr Eingabeaufforderungsoptionen, als du jemals wolltest oder brauchst, und unendliches Wachstumspotenzial durch Drittanbietererweiterungen.
```
Comment (Wolfsblvt) [inline .github/readme-de_de.md:22]: Not sure if it's a good idea to translate "Extensions", with how much that word is tied to functionality and documentation in ST. Though of course, I know the *wish* to translate as much as possible.

File: .github/readme-de_de.md [hunk 2]
```diff
@@ -0,0 +1,383 @@
...
+
+Wir haben eine [Dokumentationswebsite](https://docs.sillytavern.app/), um die meisten deiner Fragen zu beantworten und dir den Einstieg zu erleichtern.
+
+## Was ist SillyTavern?
+
+SillyTavern (oder ST für kurz) ist eine lokal installierte Benutzeroberfläche, die es dir ermöglicht, mit Textgenerations-LLMs, Bildgenerierungsmaschinen und TTS-Sprachmodellen zu interagieren.
```
Comment (Wolfsblvt) [inline .github/readme-de_de.md]:
```suggestion
SillyTavern (oder ST abgekürtz) ist eine lokal installierte Benutzeroberfläche, die es dir ermöglicht, mit Textgenerations-LLMs, Bildgenerierungsmaschinen und TTS-Sprachmodellen zu interagieren.
```

Also, what are "Bildergenerierungsmaschinen"? :D

File: .github/readme-de_de.md [hunk 3]
```diff
@@ -0,0 +1,383 @@
...
+
+Angefangen im Februar 2023 als Fork von TavernAI 1.2.8 hat SillyTavern nun über 100 Mitwirkende und 2 Jahre unabhängiger Entwicklung hinter sich und dient weiterhin als führende Software für versierte KI-Hobbyisten.
+
+## Unsere Vision
+
+1. Wir möchten die Nutzer mit so viel Nutzen und Kontrolle über ihre LLM-Eingabeaufforderungen wie möglich ausstatten. Die steile Lernkurve ist Teil des Spaßes!
```
Comment (Wolfsblvt) [inline .github/readme-de_de.md]: "LLM-Eingabeaufforderungen" sounds very weird. Might be a better way to translate that, if it has to be translated. Likely "Anweisungen". "Instruktionen" might be too much leaning into actual instruct templates.
But honestly, most German sites I've read so far do not translate too much of the LLM lingo. So it should just stay "Prompt".
```suggestion
1. Wir möchten die Nutzer mit so viel Nutzen und Kontrolle über ihre LLM-Prompts wie möglich ausstatten. Die steile Lernkurve ist Teil des Spaßes!
```

File: .github/readme-de_de.md [hunk 4]
```diff
@@ -0,0 +1,383 @@
...
+2. Wir bieten keine Online- oder gehosteten Dienste an, noch verfolgen wir programmgesteuert Benutzerdaten.
```
Comment (Wolfsblvt) [inline .github/readme-de_de.md]:
```suggestion
2. Wir bieten weder Online- oder gehosteten Dienste an, noch verfolgen wir programmgesteuert Benutzerdaten.
```
fix grammar

File: .github/readme-de_de.md [hunk 5]
```diff
@@ -0,0 +1,383 @@
...
+3. SillyTavern ist ein Herzensprojekt, das von einer engagierten Community von LLM-Enthusiasten unterstützt wird, und wird immer kostenlos und Open Source sein.
+
+## Zweige
```
Comment (Wolfsblvt) [inline .github/readme-de_de.md]:
```suggestion
## Branches
```
Even the german GitHub doc does not translate the word "Branches".

File: .github/readme-de_de.md [hunk 6]
```diff
@@ -0,0 +1,383 @@
...
+
+SillyTavern wird mit einem Zweigsystem entwickelt, um ein reibungsloses Erlebnis für alle Nutzer zu gewährleisten.
```
Comment (Wolfsblvt) [inline .github/readme-de_de.md]:
```suggestion
SillyTavern wird mit einem Zwei-Branches-System entwickelt, um ein reibungsloses Erlebnis für alle Nutzer zu gewährleisten.
```

File: .github/readme-de_de.md [hunk 7]
```diff
@@ -0,0 +1,383 @@
...
+
+* `release` -🌟 **Empfohlen für die meisten Nutzer.** Dies ist der stabilste und empfohlene Zweig, der nur aktualisiert wird, wenn wichtige Versionen veröffentlicht werden. Er ist für die Mehrheit der Nutzer geeignet. Typischerweise einmal im Monat aktualisiert.
```
Comment (Wolfsblvt) [inline .github/readme-de_de.md]:
```suggestion
* `release` -🌟 **Empfohlen für die meisten Nutzer.** Dies ist der stabilste und empfohlene Branch, der nur aktualisiert wird, wenn größere Versionen und Änderungen veröffentlicht werden. Er ist für die Mehrheit der Nutzer geeignet. Typischerweise einmal im Monat aktualisiert.
```

File: .github/readme-de_de.md [hunk 8]
```diff
@@ -0,0 +1,383 @@
...
+* `staging` - ⚠️ **Nicht für den gelegentlichen Gebrauch empfohlen.** Dieser Zweig enthält die neuesten Funktionen, kann jedoch jederzeit instabil sein. Nur für Power-User und Enthusiasten. Mehrmals täglich aktualisiert.
```
Comment (Wolfsblvt) [inline .github/readme-de_de.md]:
```suggestion
* `staging` - ⚠️ **Nicht für den Gelegenheitsgebrauch order unerfahrenen User empfohlen.** Dieser Branch enthält die neuesten Funktionen, kann jedoch jederzeit instabil sein. Nur für Power-User und Enthusiasten. Mehrmals täglich aktualisiert.
```

File: .github/readme-de_de.md [hunk 9]
```diff
@@ -0,0 +1,383 @@
...
+
+Wenn du nicht vertraut mit der Verwendung der git CLI bist oder nicht verstehst, was ein Zweig ist, mach dir keine Sorgen! Der Release-Zweig ist immer die bevorzugte Option für dich.
```
Comment (Wolfsblvt) [inline .github/readme-de_de.md]:
```suggestion
Wenn du nicht vertraut mit der Verwendung der git CLI bist oder nicht verstehst, was ein Branch ist, mach dir keine Sorgen! Der Release-Branch ist immer die bevorzugte Option für dich.
```

File: .github/readme-de_de.md [hunk 10]
```diff
@@ -0,0 +1,383 @@
...
+
+## Was brauche ich zusätzlich zu SillyTavern?
+
+Da SillyTavern nur eine Benutzeroberfläche ist, benötigst du Zugriff auf ein LLM-Backend, um Inferenz bereitzustellen. Du kannst AI Horde für sofortiges Chatten ohne weitere Einrichtung verwenden. Darüber hinaus unterstützen wir viele andere lokale und cloudbasierte LLM-Backends: OpenAI-kompatible API, KoboldAI, Tabby und viele mehr. Du kannst mehr über unsere unterstützten APIs in [der FAQ](https://docs.sillytavern.app/usage/api-connections/) lesen.
+
+### Brauche ich einen leistungsstarken PC, um SillyTavern auszuführen?
+
+Die Hardwareanforderungen sind minimal: Es läuft auf allem, was NodeJS 18 oder höher ausführen kann. Wenn du LLM-Inferenz auf deinem lokalen Rechner durchführen möchtest, empfehlen wir eine NVIDIA-Grafikkarte der 3000er-Serie mit mindestens 6 GB VRAM. Überprüfe die Dokumentation deines Backends für weitere Einzelheiten.
+
+### Vorgeschlagene Backends (nicht verbunden)
```
Comment (Wolfsblvt) [inline .github/readme-de_de.md]: lol
```suggestion
### Vorgeschlagene Backends (keine Partnerschaft oder Werbebeziehung)
```

File: .github/readme-de_de.md [hunk 11]
```diff
@@ -0,0 +1,383 @@
...
+
+* [AI Horde](https://aihorde.net/) - verwende Modelle, die von Freiwilligen gehostet werden. Erfordert keine weitere Einrichtung
+* [KoboldCpp](https://github.com/LostRuins/koboldcpp) - ein Favorit der Community, um GGUF-Modelle lokal auszuführen
+* [tabbyAPI](https://github.com/theroyallab/tabbyAPI) - eine beliebte, leichtgewichtige, lokal gehostete exl2 Inferenz-API
```
Comment (Wolfsblvt) [inline .github/readme-de_de.md]:
```suggestion
* [tabbyAPI](https://github.com/theroyallab/tabbyAPI) - eine beliebte, portable, speicherplatzoptimierte und lokal gehostete exl2 Inferenz-API
```

File: .github/readme-de_de.md [hunk 12]
```diff
@@ -0,0 +1,383 @@
...
+* [OpenRouter](https://openrouter.ai) - eine einzige API für viele Cloud-Anbieter (OpenAI, Claude, Meta Llama usw.) sowie beliebte Community-Modelle.
+
+## Fragen oder Vorschläge?
+
+### Discord-Server
+
+| [![][discord-shield-badge]][discord-link] | [Tritt unserer Discord-Community bei!](https://discord.gg/sillytavern) Erhalte Unterstützung, teile deine Lieblingscharaktere und Eingabeaufforderungen. |
```
Comment (Wolfsblvt) [inline .github/readme-de_de.md]:
```suggestion
| [![][discord-shield-badge]][discord-link] | [Tritt unserer Discord-Community bei!](https://discord.gg/sillytavern) Erhalte Unterstützung, teile deine Lieblingscharaktere und Prompts. |
```

File: .github/readme-de_de.md [hunk 13]
```diff
@@ -0,0 +1,383 @@
...
+| :---------------------------------------- | :----------------------------------------------------------------------------------------------------------------- |
+
+Oder nimm direkt Kontakt mit den Entwicklern auf:
+
+* Discord: cohee, rossascends, wolfsblvt
+* Reddit: [/u/RossAscends](https://www.reddit.com/user/RossAscends/), [/u/sillylossy](https://www.reddit.com/user/sillylossy/), [u/Wolfsblvt](https://www.reddit.com/user/Wolfsblvt/)
+* [Erstelle ein GitHub-Problem](https://github.com/SillyTavern/SillyTavern/issues)
```
Comment (Wolfsblvt) [inline .github/readme-de_de.md]:
```suggestion
* [Erstelle ein GitHub-Issue](https://github.com/SillyTavern/SillyTavern/issues)
```
Same here. Cross-check GitHubs German docs, if you want to know the wordings they use. I would not deviate from those. They still use "Issue".

File: .github/readme-de_de.md [hunk 14]
```diff
@@ -0,0 +1,383 @@
...
+
+### Ich mag dein Projekt! Wie kann ich beitragen?
```
Comment (Wolfsblvt) [inline .github/readme-de_de.md]:
```suggestion
### Ich mag dieses Projekt! Wie kann ich beitragen?
```
Neither singular no plural is really accurate. Circumventing the use is the easiest solution.

File: .github/readme-de_de.md [hunk 15]
```diff
@@ -0,0 +1,383 @@
...
+
+1. Sende Pull-Requests. Lerne, wie du beitragen kannst: [CONTRIBUTING.md](../CONTRIBUTING.md)
+2. Sende Funktionser Vorschläge und Problemmeldungen unter Verwendung der bereitgestellten Vorlagen.
```
Comment (Wolfsblvt) [inline .github/readme-de_de.md]:
```suggestion
2. Sende Feature Requests und Issues unter Verwendung der bereitgestellten Vorlagen.
```

File: .github/readme-de_de.md [hunk 16]
```diff
@@ -0,0 +1,383 @@
...
+3. Lies diese gesamte README-Datei und überprüfe zuerst die Dokumentationswebsite, um doppelte Probleme zu vermeiden.
```
Comment (Wolfsblvt) [inline .github/readme-de_de.md]:
```suggestion
3. Lies diese gesamte README-Datei und überprüfe zuerst die Dokumentationswebsite, um doppelte Issues zu vermeiden.
```

File: .github/readme-de_de.md [hunk 17]
```diff
@@ -0,0 +1,383 @@
...
+
+## Screenshots
+
+<img width="500" alt="image" src="https://github.com/user-attachments/assets/9b5f32f0-c3b3-4102-b3f5-0e9213c0f50f">
+<img width="500" alt="image" src="https://github.com/user-attachments/assets/913fdbaa-7d33-42f1-ae2c-89dca41c53d1">
+
+## Charakterkarten
+
+SillyTavern basiert auf dem Konzept der "Charakterkarten". Eine Charakterkarte ist eine Sammlung von Eingabeaufforderungen, die das Verhalten des LLM festlegen und erforderlich sind, um anhaltende Gespräche in SillyTavern zu führen. Sie funktionieren ähnlich wie ChatGPTs GPTs oder Poes Bots. Der Inhalt einer Charakterkarte kann alles sein: ein abstraktes Szenario, ein Assistent, der für eine bestimmte Aufgabe maßgeschneidert ist, eine berühmte Persönlichkeit oder ein fiktiver Charakter.
```
Comment (Wolfsblvt) [inline .github/readme-de_de.md]:
```suggestion
SillyTavern basiert auf dem Konzept der "Charakterkarten". Eine Charakterkarte ist eine Sammlung von Prompts, die das Verhalten des LLM festlegen und erforderlich sind, um persistente Gespräche in SillyTavern zu führen. Sie funktionieren ähnlich wie ChatGPT's GPTs oder Poe's Bots. Der Inhalt einer Charakterkarte kann alles sein: ein abstraktes Szenario, ein Assistent, der für eine bestimmte Aufgabe maßgeschneidert ist, eine berühmte Persönlichkeit oder ein fiktiver Charakter.
```

File: .github/readme-de_de.md [hunk 18]
```diff
@@ -0,0 +1,383 @@
...
+
+Das Namensfeld ist der einzige erforderliche Eingabewert für die Charakterkarte. Um ein neutrales Gespräch mit dem Sprachmodell zu beginnen, erstelle eine neue Karte, die einfach "Assistent" genannt wird, und lasse die restlichen Felder leer. Für einen thematischeren Chat kannst du dem Sprachmodell verschiedene Hintergrundinformationen, Verhaltensweisen und Schreibmuster sowie ein Szenario geben, um das Gespräch zu beginnen.
```
Comment (Wolfsblvt) [inline .github/readme-de_de.md]:
```suggestion
Das Namensfeld ist der einzige erforderliche Eingabewert für die Charakterkarte. Um ein neutrales Gespräch mit dem LLM zu beginnen, erstelle eine neue Karte, die einfach "Assistent" genannt wird, und lasse die restlichen Felder leer. Für einen thematischeren Chat kannst du dem LLM verschiedene Hintergrundinformationen, Verhaltensweisen und Schreibmuster sowie ein Szenario geben, um das Gespräch zu beginnen.
```

File: .github/readme-de_de.md [hunk 19]
```diff
@@ -0,0 +1,383 @@
...
+
+Um ein schnelles Gespräch zu führen, ohne eine Charakterkarte auszuwählen, oder um einfach die LLM-Verbindung zu testen, gib einfach deine Eingabeaufforderung in die Eingabezeile auf dem Willkommensbildschirm ein, nachdem du SillyTavern geöffnet hast. Bitte beachte, dass solche Chats vorübergehend sind und nicht gespeichert werden.
```
Comment (Wolfsblvt) [inline .github/readme-de_de.md]:
```suggestion
Um ein schnelles Gespräch zu führen, ohne eine Charakterkarte auszuwählen, oder um einfach die LLM-Verbindung zu testen, gib einfach dein Prompt in die Eingabezeile auf dem Willkommensbildschirm ein, nachdem du SillyTavern geöffnet hast. Bitte beachte, dass solche Chats vorübergehend sind und nicht gespeichert werden.
```

File: .github/readme-de_de.md [hunk 20]
```diff
@@ -0,0 +1,383 @@
...
+
+Um eine allgemeine Vorstellung davon zu bekommen, wie man Charakterkarten definiert, sieh dir die Standardfigur (Seraphina) an oder lade ausgewählte von der Community erstellte Karten im Menü "Erweiterungen & Assets herunterladen" herunter.
```
Comment (Wolfsblvt) [inline .github/readme-de_de.md]:
```suggestion
Um eine allgemeine Vorstellung davon zu bekommen, wie man Charakterkarten definiert, sieh dir die mitgelieferte Charakterkarte (Seraphina) an oder lade ausgewählte von der Community erstellte Karten im Menü "Erweiterungen & Assets herunterladen" herunter.
```

File: .github/readme-de_de.md [hunk 21]
```diff
@@ -0,0 +1,383 @@
...
+
+## Hauptmerkmale
```
Comment (Wolfsblvt) [inline .github/readme-de_de.md]:
```suggestion
## Wichtigste Features
```
I don't think most places actually translate "Features". *Merkmale* is not the correct wording for sure.

File: .github/readme-de_de.md [hunk 22]
```diff
@@ -0,0 +1,383 @@
...
+
+* Erweiterte Texteinstellungen mit vielen von der Community erstellten Voreinstellungen
```
Comment (Wolfsblvt) [inline .github/readme-de_de.md]:
```suggestion
* Erweiterte Text-Generierungs-Einstellungen mit vielen von der Community erstellten mitgelieferten Einstellungen (Presets)
```

File: .github/readme-de_de.md [hunk 23]
```diff
@@ -0,0 +1,383 @@
...
+* Unterstützung für World Info: erstelle reichhaltige Lore oder speichere Tokens auf deiner Charakterkarte
```
Comment (Wolfsblvt) [inline .github/readme-de_de.md]: This translation is just wrong.
```suggestion
* Unterstützung für World Info: Erstelle reichhaltige Lore oder reduziere die Tokens in deiner Charakterkarte
```

File: .github/readme-de_de.md [hunk 24]
```diff
@@ -0,0 +1,383 @@
...
+* Gruppenchats: Multi-Bot-Räume für Charaktere, die mit dir und/oder untereinander sprechen
+* Reichhaltige UI-Anpassungsoptionen: Themenfarben, Hintergrundbilder, benutzerdefiniertes CSS und mehr
```
Comment (Wolfsblvt) [inline .github/readme-de_de.md]: I think Themes should not be translated either.
```suggestion
* Reichhaltige UI-Anpassungsoptionen: Themes zur Farbenwahl, Hintergrundbilder, benutzerdefiniertes CSS und mehr
```

File: .github/readme-de_de.md [hunk 25]
```diff
@@ -0,0 +1,383 @@
...
+* Benutzer-Personas: Lass die KI ein wenig über dich wissen, um die Immersion zu erhöhen
+* Eingebaute RAG-Unterstützung: Füge Dokumente zu deinen Chats hinzu, auf die die KI verweisen kann
+* Umfangreiches Chat-Befehlsunterstützungssystem und eigene [Scripting-Engine](https://docs.sillytavern.app/usage/st-script/)
```
Comment (Wolfsblvt) [inline .github/readme-de_de.md]:
```suggestion
* Umfangreiches "Chat-Befehle"-System und eigene [Scripting-Engine](https://docs.sillytavern.app/usage/st-script/)
```

File: .github/readme-de_de.md [hunk 26]
```diff
@@ -0,0 +1,383 @@
...
+
+## Erweiterungen
+
+SillyTavern hat Unterstützung für Erweiterungen.
```
Comment (Wolfsblvt) [inline .github/readme-de_de.md]:
```suggestion
SillyTavern unterstützt Erweiterungen.
```

File: .github/readme-de_de.md [hunk 27]
```diff
@@ -0,0 +1,383 @@
...
+
+* Emotionale Ausdrucksformen von Charakteren (Sprites)
+* Automatische Zusammenfassung des Chatverlaufs
+* Automatische UI- und Chat-Übersetzung
+* Bildgenerierung mit Stable Diffusion/FLUX/DALL-E
+* Text-to-Speech für KI-Antwortnachrichten (über ElevenLabs, Silero oder die TTS-Funktion des Betriebssystems)
+* Websuchfunktionen zum Hinzufügen zusätzlicher realer Kontexte zu deinen Eingabeaufforderungen
+* Viele weitere sind im Menü "Erweiterungen & Assets herunterladen" verfügbar.
+
+Tutorials zur Nutzung findest du in den [Docs](https://docs.sillytavern.app/).
```
Comment (Wolfsblvt) [inline .github/readme-de_de.md]:
```suggestion
Tutorials zur Nutzung findest du in der [Dokumentation](https://docs.sillytavern.app/).
```

File: .github/readme-de_de.md [hunk 28]
```diff
@@ -0,0 +1,383 @@
...
+
+# ⌛ Installation
+
+> \[!WARNING]
+>
+> * INSTALLIERE NICHT IN EINEM VON WINDOWS GESTEUERTEN ORDNER (Programme, System32 usw.).
```
Comment (Wolfsblvt) [inline .github/readme-de_de.md]:
```suggestion
> * INSTALLIERE NICHT IN EINEM VON WINDOWS KONTROLLIERTEN ORDNER (Programme, System32 usw.).
```

**Review by Wolfsblvt — CHANGES_REQUESTED** [review]
Stopped at the "Installation" section. This is taking too much of my time.

Did you use machine translation for this without manually vetting and fixing all the issues and mis-translations and made sure that actual technical wording that made sense stays english?

If I continue going through this, I might've been faster to translate this with ChatGPT and manually fixing myself.

I am sorry, but I think the quality is not much better than what browser-internal translation tools provide out of the box.

Can you update this PR to actual reasonable translations? Otherwise I'd think we can just close this and let Germans read the actual english readme.

File: .github/readme-de_de.md (hunk 1 above)
Comment (X00LA) [reply .github/readme-de_de.md:22]: I don't know what the problem is? The other translations are also ***complete*** translations, so ***why*** should be german any ***different***?

**Comment by X00LA** [comment]
Checking the file for any spelling issues today!
I'll let you know as soon as it's done.

**Comment by X00LA** [comment]
Fixed all issues and now it is a 1:1 tanslation from the original.
Fixed spelling issues and wrong text blocks.

**Comment by X00LA** [comment]
Now everything should be correct.
I hope it satisfies your requirements?

**Comment by Cohee1207** [comment]
@Wolfsblvt are we good here?

**Comment by Wolfsblvt** [comment]
Well, just from a quick glance I still saw like a handful mentions of „Zweig“, which is the German word for tree branch.

I mean, merge it if you want. I wouldn‘t consider this a good translation for a tech tool though.

**Comment by X00LA** [comment]
I overlooked this "Zweig" (6 in one Paragraph that I overlooked) when proofreading and have now changed it. And "Zweig" does NOT just mean tree branch in German! It has, like in English, several meanings! Wolfsblvt is harping on about trivialities here.
Otherwise, this translation is a 1 to 1 translation.
I don't understand what there is to complain about?!
"I wouldn't consider this a good translation for a tech tool though." So "Wolfsblvt" then the original is probably not good enough either?
Send my translation through a translation tool of your choice and you should get an almost identical English version!
But if it's not good enough, then I have no problem with it not being used and instead Wolfsblvt creating one to meet her requirements.

**Comment by Wolfsblvt** [comment]
Well, "Zweig" might be uses in context like ancestry trees or branching roads. Sure. Still not the word most common sites and tools use to translate "branches".

That's what I meant with *decent translation for a tech tool*. Your translation is good - besides a few issues that you mostly fixed.  
It translates IT and tech terms into German that many people in that area would not translate though. That was my first big complain here.

I am biased though. Even as a native German speaker, I don't like reading anything tech in German. Makes me cringe.  
I can still see the use for people being less fluent in reading English, so a German readme for sure is a good idea. It's just that we can likely debate for weeks which words should not be translated, because people should have the most basic knowledge in English terms that are setting the precedents what stuff is, for sure.  
And because I know I am biased, I cross-referenced with the official GitHub documentation in German for many things. Which supported me in my stance.

I have to admit, I don't care enough about this to take another hour to read through this and try to find some suggestions on my side.  
You seem to be far more the target audience for this than me anyway.

(I have he/him pronouns in my bio BTW (: )

File: .github/readme-de_de.md (hunk 1 above)
Comment (Wolfsblvt) [reply .github/readme-de_de.md:22]: I cannot read the other translations, as I am not fluent in them. If it‘s translated in the UI for german, it makes sense to translate this here in the readme too, yeah.

File: .github/readme-de_de.md (hunk 10 above)
Comment (Wolfsblvt) [reply .github/readme-de_de.md]: The original intent of that heading was not about connecting to the backends. It‘s affiliate.

Heißt wir listen hier mögliche Backends auf, aber wir arbeiten nicht mit denen Zusammen oder kriegen irgendwas dafür die hier zu listen. Wieder so eine schöne Stelle, wo es kein passendes deutsches Wort gibt. Aber „verbunden“ ist es nicht.

**Comment by X00LA** [comment]
When I translate, I always assume that people who don't understand English will read it, so I always translate everything so that people can understand it.
I would like to apologize for the "she". I wasn't in your bio and was just guided by the avatar picture.

**Comment by Wolfsblvt** [comment]
Yeah, that‘s where we likely just differ in opinion. I prefer to use terms that are known in that area/niche, and I‘d focus on re-using the same terms, translations (and also not-translated words) than bigger sites, industry leaders and Co.
That‘s why I referred to the German GitHub documentation when I suggested to not translate „Branch“, „Issue“, „Pull Request“ and similar things. They don‘t either, for people who actually read the documentation of how GitHub works.
But yeah, just a different take here.

There is more stuff. A few things I commented on that you neither replied to nor fixed.
Some are debatable or opinionated, sure. But Seraphina is our default character that gets shipped with ST. You translated characters/cards as „Charakterkarten“. Then „Standardfigur“ is not the correct translation here.

But I am here because Cohee asked me for feedback - likely because I am German and also know ST well enough. He should‘ve known I am pedantic too.
So yeah. 
Maybe I was the wrong guy to ask for this.

**Comment by X00LA** [comment]
I've gone through everything now, so there shouldn't be any more problems. I hope...

**Review by Wolfsblvt — APPROVED** [review]

**Review by Cohee1207 — APPROVED** [review]

**Pull request merged**